DP03_05_Near-Earth_Objects.ipynb

```diff
@@ -0,0 +1,1395 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "749b0ddf",
+   "metadata": {},
+   "source": [
+    "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\"> \n",
+    "<br><b>CET Template Notebook</b> <br>\n",
+    "Contact author(s): <i>Sarah Greenstreet</i> <br>\n",
+    "Last verified to run: <i>2024-03-05</i> <br>\n",
+    "LSST Science Pipelines version: Weekly <i>2024_04</i> <br>\n",
+    "Container Size: <i>medium</i> <br>\n",
+    "Targeted learning level: <i>beginner</i> <br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9da1a210-d858-42fe-8591-570965b8be1a",
+   "metadata": {},
+   "source": [
+    "**Description:** _An introduction to near-Earth objects (NEOs) in the DP0.3 dataset, including illustrations of the four NEOs sub-populations: Amors, Apollos, Atens, and Atiras._"
```

Comment:
these are italicized, the dp02 nb are not, do we want to be consistent?

Reply:
Italics removed for consistency.